providers/summary/clearlydefined.js

```diff
@@ -181,28 +189,44 @@ class ClearlyDescribedSummarizer {
     setIfValue(result, 'licensed.declared', license)
   }
 
-  addPodData(result, data) {
+  addPodData(result, data, coordinates) {
     setIfValue(result, 'described.releaseDate', extractDate(data.releaseDate))
     setIfValue(result, 'described.projectWebsite', get(data, 'registryData.homepage'))
     const license = get(data, 'registryData.license')
     if (license) {
       setIfValue(result, 'licensed.declared', SPDX.normalize(typeof license === 'string' ? license : license.type))
     }
+
+    const homepage = get(data, 'registryData.homepage')
+    const revision = get(data, 'registryData.sourceInfo.revision')
+
+    const registry = buildRegistryUrl(coordinates)
+    const version = homepage && revision ? `${homepage}/tree/${revision}` : null
+    const download = homepage && revision ? `${homepage}/archive/${revision}.zip` : null
```

Comment:
This only works when the homepage is on github

This one for example would not work, https://api.clearlydefined.io/definitions/pod/cocoapods/-/crashlytics/3.12.0

Reply:
take a look at https://github.com/clearlydefined/crawler/blob/master/providers/fetch/podFetch.js#L75 for how we resolve it.

Reply:
Ah, thanks, i hadn't any example for that case.
Looking at the harvested data of the example, looks like that we can't link to a specific version URL, since it doesn't exists.

Reply:
that's what I found too :( that's okay